target/linux/ramips/image/Makefile

```diff
@@ -117,6 +134,12 @@ define Build/sercom-footer
 	$(call Build/sercom-seal,-f)
 endef
 
+define Build/sercomm-kernel
```

Comment:
Common spelling seams to be with one "m" ...?

Reply:
Common spelling is with double m:
1. OpenWrt code - https://github.com/openwrt/openwrt/search?q=sercomm
2. Company name - https://www.sercomm.com/contpage.aspx?langid=1&type=info&L1id=1&L2id=1

Nevertheless, if I need to fix double m, please, let me know your decision.